infrastructure/lib/constructs/lambda-fn.ts

```diff
@@ -132,6 +139,24 @@ export class ApiFlexFunction extends cdk.Construct {
       entry: props.handlerPath,
       vpc: props.lambdaVpc,
       ...props.functionPropsOverride,
+      bundling: {
```

Comment:
Any reason why we can't define a global object for this block since we re-use it in 3 places?

Reply:
One of them `ApiFunction` is legacy and probably should have been deleted in #464. We _might_ be able to. Let me play with it.

Reply:
I think that mostly I have de-duplicated this. Once by accepting `ApiFunction` is deprecated and I _hope_ we break existing uses of it. The other by trying to have `database.ts` use the Function construct created in `lambda-fn.ts`.